bets.csv

```diff
@@ -3,5 +3,6 @@ Les,04-05-2020
 Bruna,8-8-2020
 Eefje,01-07-2020
 Sander,5-12-2020
-Chiel,1-10-2020
-Giwan,01-04-2020
\ No newline at end of file
+Chiel2,1-10-2020
+Giwan,01-04-2020
+Chiel,31-07-2020

```

Comment:
Hi Chiel, thank you for your contribution.

Please next time refer to the [submission requirements](https://www.youtube.com/watch?v=dQw4w9WgXcQ) before you submit a pull request.

As to this specific pull request, I have a point of critique: As you have clearly noted, the name "Chiel" is ambiguous, so please feel free to change it.